packages/@okta/vuepress-site/docs/reference/registration-hook/index.md

```diff
@@ -158,15 +158,21 @@ Registrations are allowed by default, so setting a value of `ALLOW` for the `act
 
 ### error
 
-See [error](/docs/concepts/inline-hooks/#error) for general information on the structure to use for the `error` object.
+See [error](/docs/concepts/inline-hooks/#error) for general information about the structure to use for the `error` object.
 
-For the Registration Inline Hook, the `error` object provides a way of displaying an error message to the end user who is trying to register. If you're using the Okta Sign-In Widget for Profile Enrollment, and have not customized its error handling behavior, only the `errorSummary` of the first `errorCauses` object that your external service returns appears to the end user.
+> **Note:** If you include an error object in your response, no commands are executed and the registration fails. This holds true even if the top-level `errorSummary` and the `errorCauses` objects are omitted.
 
-If you don't return any value for that `errorCauses` object and deny the user's registration attempt through the `commands` object in your response to Okta, the following generic message appears to the end user: "Registration cannot be completed at this time".
+For the Registration Inline Hook, the `error` object provides a way of displaying an error message to the end user who is trying to register.
 
-If you don't return an `error` object at all and the registration is denied, the following generic message appears to the end user: "Registration denied".
+* If you're using the Okta Sign-In Widget for Profile Enrollment, only the `errorSummary` messages of the `errorCauses` objects that your external service returns appear as inline errors, given the following:
 
-> **Note:** If you include an error object in your response, no commands will be executed and the registration will fail. This holds true even if the top-level `errorSummary` and the `errorCauses` objects are omitted.
+   * You don't customize the error handling behavior of the Widget.
+   * The attributes are in the correct location of the `errorCauses` object. See the [sample JSON payload of the request](#sample-json-payload-of-request).
```

Comment:
I'm assuming the links is supposed to go to the `sample JSON payload RESPONSE` (On a separate item of style, I noticed you lower-capped the first letter of the heading you linked to because it's part of the sentence; however, our style appears to be to include the capitalization of the hyperlinked section, even within a sentence. I pinged Susan about it thought we might discuss)

Reply:
No need. I agree.